pyspelling/util/__init__.py

```diff
@@ -1,5 +1,5 @@
 """Utilities."""
-from __future__ import unicode_literals
+from __future__ import unicode_literals, print_function
```

Comment:
We honestly don't need print_function as we only support python3. To be honest, importing of __future__ can be completely removed.